package.json

```diff
@@ -58,11 +58,12 @@
   },
   "homepage": "https://github.com/bbc-news/simorgh#readme",
   "dependencies": {
-    "@bbc/psammead-copyright": "^0.2.0",
+    "@bbc/psammead-brand": "^0.1.1",
     "@bbc/psammead-headings": "^0.1.2",
+    "@bbc/psammead-paragraph": "^0.1.3",
+    "@bbc/psammead-copyright": "^0.2.0",
```

Comment:
Can you make this list alphabetical? Thanks!

Reply:
@ChrisBAshton I have addressed this comment